community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,460 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aqua-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: Security
+    description: The Aqua Security Operator runs within a Kubernetes or Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components.
+    certified: "false"
+    containerImage: nissimaqua/aqua-operator:0.0.1
```

Comment:
Is this meant to use a personal docker username rather than an org?

Reply:
yes, because the organization repository need user name and password and your ci wouldn't work with private registry.

Reply:
The organization could have a public image for this operator, which wouldn't require a password

Reply:
Ok, i will create a public docker repository of the company, thanks you